aml-development/ozp-backend
# feature(coverage): Increase coverage includes storefront feature listing sort

AMLNG-855   (storefront feature listing sort)

**Description**     
Update Featured Listings to be displayed as the most recently featured Listing to be displayed first in the list (LIFO)  

**Acceptance Criteria**   
The last Listing featured will be the first Listing displayed in the Featured Listing section
 
**How to test code**    
From the StoreFront
  -Select a Listing
  -Select the Listing Administration tab
  -Feature a Listing by turning 'on' the Featured  toggle button
  -Refresh the page
  **Expected Result**: The selected listing will appear first in the the feature listing section


**Please Review**     
@aml-development/ozp-backend-team    

**Checklist**
